autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -399,6 +399,13 @@ def fit(train_data,
                 Note: The file containing your `FeatureGenerator` class must be imported into current Python session in order to use a custom class.
             feature_generator_kwargs : dict, default={}
                 Keyword arguments to pass into the `FeatureGenerator` constructor.
+                Valid `AutoMLFeatureGenerator` kwargs:
+                    enable_nlp_vectorizer_features : bool, default = True
+                        If True, the vectorizer argument value is used to generate 'nlp_ngram' features from text features if present.
+                    enable_nlp_ratio_features : bool, default = True
+                        If True, generate 'nlp_special' features from text features if present.
```

Comment:
why is the argument named `nlp_ratio_features` but the docstring refers to `nlp_special` features?  It would also be good to (briefly) list what these features are (perhaps in English rather than variable-names)

Reply:
I had the same urge to change it originally. Reason I didn't was because of backwards compatibility, but I think very few people probably set this value beforehand and it was essentially internal, so its probably safe to change. Done. Also improved description.